Simonas B88 - Login (#274)
PrestaShop/prestashop-specs: content/1.7/back-office/login-listing.md

---
title: Login into Back-Office
weight: 1
---
# LOGIN

> [Login UI](static/login.png)

Reviewer: ```suggestion
![Login UI](static/login.png)
```

This UI is the main feature to authorize the Prestashop Back-Office administrator, and to log-in to administrate and maintain all the Prestashop Back-Office  features and functionalities. This page contains only one main element - the log in form.

## Top UI Section

- The top section represents the Prestashop main title name and the Prestashop version, that the administrator is accessing.

## Middle UI Section

> 

Author: @MatShir I didn't specify that certain version number, thought, It should be described globally? So I have to specify Prestashop 1.7.8.0 in this spec? :)

> 

Reviewer: just need to write that is where the version is mentioned

- **Logo** - the official Preston seabird logo of Prestashop.
- **Shop name** - the shop name, that was defined during the installation process of Prestashop application.

> - **Email address input** - the specific superadmin profile email input text field. The email is the exact that was defined during the installation process of Prestashop. It must be used at the first log-in attempt. 

Reviewer: ```suggestion
- **Email address** - email input - Email from a SuperAdmin or Employees acount must be used to log-in. 
```
The mentioned part should be more for the installation process.

> - **Password input** - the specific superadmin profile password input text field. The password is the exact that was defined during the installation process of Prestashop. It must be used at the first log-in attempt. 

Reviewer: ```suggestion
- **Password input** -  Password of the email account above from a SuperAdmin or Employees must be used to log-in. 
```

> - **Login CTA button** - the button, that executes the logging in action, the user then is instantly redirected to the default main dashboard page.

Reviewer: It is missing the error cases. 
What happens if the user doesn't use an email from a SuperAdmin or an Employee account?
What happens if the user doesn't use a password from a SuperAdmin or an Employee account?
Does the form warn the user if the email is formatted as an email?

Author: > What happens if the user doesn't use an email from a SuperAdmin or an Employee account? What happens if the user doesn't use a password from a SuperAdmin or an Employee account? 
It is always the same message `The employee does not exist, or the password provided is incorrect.`

> Does the form warn the user if the email is formatted as an email?

Yes, instant red error message appears, about email, that it is not valid

> - **Login CTA button** - the button, that executes the logging in action, the user then is instantly redirected to the default main dashboard page.

Reviewer: ```suggestion
- **Login button** - Allows access to the back office after verifying that the user is authorized to log-in.
```

> - **Stay logged in checkbox** - this checkbox and its function is common in many logging-in submission forms. Once the checkbox is marked, the browser will remember the logging-in status, and the administrator will not be required to input email and password in the next attempt of logging-in. The administrator will be authenticated and redirected to the dashboard instantly.

Reviewer: ```suggestion
- **Stay logged in checkbox** - The browser will remember the logging-in status, and the administrator will not be required to input email and password in the next attempt of logging-in. The administrator will be authenticated and redirected to the dashboard instantly .
```

> - **I forgot my password** - specific link, in case administrator forgets the Prestashop authentication password. Once clicked, the UI instantly flips to the new UI which contains the email input. After submitting the email there, a link will be generated and sent to the defined email.

Reviewer: What does the email contains? What can you do? Reset the password?

Author: updated with the information
